Mathlib/Data/Matrix/Invertible.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Eric Wieser
```

Comment:
Feel free to add yourself as an author here, @4hma4d!

Reply:
Thanks!